libs/core/langchain_core/vectorstores/base.py

```diff
@@ -568,7 +569,7 @@ def similarity_search_with_relevance_scores(
                 if similarity >= score_threshold
             ]
             if len(docs_and_similarities) == 0:
-                warnings.warn(
+                logger.warning(
```

Comment:
From https://docs.python.org/3/howto/logging.html#when-to-use-logging :
> [warnings.warn()](https://docs.python.org/3/library/warnings.html#warnings.warn) in library code if the issue is avoidable and the client application should be modified to eliminate the warning
A logger’s [warning()](https://docs.python.org/3/library/logging.html#logging.Logger.warning) method if there is nothing the client application can do about the situation, but the event should still be noted